docs/src/examples/lorenz_example.md

```diff
@@ -1,26 +1,289 @@
+For very small values of ``F``, the solutions $x_i$ decay to $F$ after the initial transient feature.
+For moderate values of ``F``, the solutions are periodic, and for larger values of ``F``, the system is chaotic.
+The solution variance is a function of the forcing magnitude.
+Variations in the base state as a function of time can be imposed through a time-dependent forcing term ``F(t)``.
 
-- `GModel.jl` Contains the forward map. The inputs should be the so-called free parameters we are interested in learning, and the output should be the measured data
-- The example script which contains the inverse problem setup and solve
+A temporal forcing term is defined
+```math
+F = F_s + A \sin(\omega t),
+```
+with steady-state forcing ``F_s``, transient forcing amplitude ``A``, and transient forcing frequency ``\omega``.
+The total forcing ``F`` must be within the chaotic regime of L96 for all time given the prescribed $N$.
 
-## The structure of the example script
-First we create the data and the setting for the model
-1. Set up the forward model.
-2. Construct/load the truth data. Store this data conveniently in the `Observations.Observation` object
+The L96 dynamics are solved with RK4 integration.
 
-Then we set up the inverse problem
-3. Define the prior distributions. Use the `ParameterDistribution` object
-4. Decide on which `process` tool you would like to use (we recommend you begin with `Invesion()`). Then initialize this with the relevant constructor
-5. initialize the `EnsembleKalmanProcess` object
+# Structure
 
-Then we solve the inverse problem, in a loop perform the following for as many iterations as required:
-7. Obtain the current parameter ensemble
-8. Transform them from the unbounded computational space to the physical space
-9. call the forward map on the ensemble of parameters, producing an ensemble of measured data
-10. call the `update_ensemble!` function to generate a new parameter ensemble based on the new data
+The example is structured with two distinct components: 1) L96 dynamical system solver; 2) calibrate-emulate sample code. Each of these are described below.
+
+Forward model: The forward mapping from input parameters to output statistics of the L96 system is solved using the `GModel.jl` code, which runs the L96 model across different input parameters ``\theta``. The source code for the L96 system solution is within the `GModel_common.jl` code. 
+
+Calibrate: The main code is located in `calibrate.jl` which provides the functionality to run the L96 dynamical system (within the `GModel.jl` code), extract time-averaged statistics from the L96 states, and use the time-average statistics for calibration.
+
+Emulate-sample: The main code is located in `emulate_sample.jl` which provides the functionality to run the L96 dynamical system, extract time-averaged statistics from the L96 states, and use the time-average statistics for emulation and sampling (uncertainty quantification).
+
+# Inputs
```

Comment:
The whole code is covered by this title, maybe we should call it "Walkthrough of the code?

Reply:
could you add one more code snippet for the packages loaded?